less/components/jtable.less

```diff
@@ -0,0 +1,96 @@
+//--------------------------------------------------------------------------
+//
+//  JTable and related components
+//
+//--------------------------------------------------------------------------
+
+//--------------------------------------
+//  Vars / Mixins
+//--------------------------------------
+
+@table-left-right-padding: 24px;
+@table-column-spacing: 32px;
+// !!!IMPORTANT!!! Don't change the above without changing the constants in JTable.jsx to match!
+
+// FIXME: We need to sort out the mess of different colours we've got all over the place
+@table-border-color: #e6eaee;
+@table-header-color: rgba(53, 64, 82, 0.5);
+@table-hover-bg: #fafbfd;
```

Comment:
@cliffmeyers to track the pallette you mean? ie @brody may have something on that.